embassy-stm32/src/timer/low_level.rs

```diff
@@ -235,6 +235,11 @@ impl<'d, T: CoreInstance> Timer<'d, T> {
         self.regs_core().cnt().write(|r| r.set_cnt(0));
     }
 
+    /// get the capability of the timer
+    pub fn get_bits(&self) -> TimerBits {
```

Comment:
could you make this `fn bits()`? the convention in Rust is getters don't have `get_` prefix.